apps/coordinator/ts/scheduler/__tests__/scheduler.service.test.ts

```diff
@@ -162,20 +172,173 @@ describe("SchedulerService", () => {
 
       expect(isPollTallied).toBe(false);
     });
+  });
+
+  describe("finalizePoll", () => {
+    beforeEach(() => {
+      jest.clearAllMocks();
+
+      (proofService.merge as jest.Mock).mockReset().mockResolvedValue({});
+      (proofService.generate as jest.Mock).mockReset().mockResolvedValue({});
+      (proofService.submit as jest.Mock).mockReset().mockResolvedValue({});
```

Comment:
Why do you use `mockReset` here if you have `clearAllMocks` above and use `mockResolvedValue` after?

Reply:
I remember the tests not passing because some of the mocks were not being deleted so I added this `mockReset()` calls. I tried it without these lines and the tests seem to work so I will remove them